src/views/Stack/Card.tsx

```diff
@@ -412,18 +412,25 @@ export default class Card extends React.Component<Props> {
       cond(
         eq(this.props.current, isVisible),
         call(
-          [this.didMovementHappen, this.isVisible],
-          ([didMovementHappen]: ReadonlyArray<Binary>) => {
+          [this.didMovementHappen, isVisible],
```

Comment:
We need to handle a case when we release not components either on the beginning or end of animation